daft/io/_parquet.py

```diff
@@ -55,6 +56,8 @@ def read_parquet(
 
     file_format_config = FileFormatConfig.from_parquet_config(ParquetSourceConfig())
     if use_native_downloader:
+        if io_config is None:
```

Comment:
Is there an advantage to creating an `io_config` vs leaving it as None?

Reply:
Ah, no you're right, we should propagate the `None` in this case.